core/src/test/java/org/mskcc/cbio/portal/scripts/TestImportTabDelimData.java

```diff
@@ -328,6 +330,7 @@ public void testImportmRnaData2() throws Exception {
         geneticProfile.setDatatype("CONTINUOUS");
         geneticProfile.setProfileName("MRNA Data");
         geneticProfile.setProfileDescription("mRNA Data");
+        geneticProfile.setReferenceGenomeId(1);
```

Comment:
here

Reply:
reference_id assigned not auto generated. will not change.